lib/pool.js

```diff
@@ -6,6 +6,7 @@ const {
 } = require('./core/errors')
 const FixedQueue = require('./core/node/fixed-queue')
 
+const { kTLSSession } = require('./core/symbols')
```

Comment:
`const kTLSSession = Symbol('tlsSession')`

Reply:
I'd like to avoid confusion for people looking at `Pool` so that they don't think the implementation depends on internals.

Reply:
I'm not sure I follow. I **must** use the symbol present in our symbol registry, otherwise, the pool will not access the correct cache.